examples/ota-provider-app/linux/README.md

```diff
@@ -52,6 +52,26 @@ An example of the `--otaImageList` file contents:
 }
 ```
 
+## Access Control Requirements
+
+Commissioner or Administrator SHOULD install necessary ACL entries at
+commissioning time or later to enable processing of QueryImage commands from OTA
+Requestors on their fabric, otherwise that OTA Provider will not be usable by
+OTA Requestors. For testing purpose, below is an example of how to write the ACL
+attribute with just one entry: Administer privilege, CASE authentication mode,
+wildcard subject (all nodes), wildcard target (all clusters on every endpoint).
```

Comment:
Why would one ever do this for OTA Provider?  Wouldn't it at the very least set the cluster to "OTA Provider" even if leaving everything else as wildcards?  That would actually be a sane-ish ACL to use and then would not need the "real world scenarios" warning.

Reply:
I was following @mlepage-google advice and leaving this as open as possible for testing. I could make it stricter to only the OTA Provider cluster as target.

Reply:
Agree with @bzbarsky-apple: For the OTA provider, privilege only needs to be *Operate*, and only cluster OTA Provider. This is more realistic and safer.

Reply:
Updated the example